perfkitbenchmarker/disk.py

```diff
@@ -73,8 +162,8 @@ def __init__(self, disk_size=None, disk_type=None,
 
   def ApplyFlags(self, flags):
     """Applies flags to the DiskSpec."""
-    self.disk_size = flags.scratch_disk_size or self.disk_size
-    self.disk_type = flags.scratch_disk_type or self.disk_type
+    self.disk_size = flags.data_disk_size or self.disk_size
+    self.disk_type = flags.data_disk_type or self.disk_type
     self.num_striped_disks = flags.num_striped_disks or self.num_striped_disks
```

Comment:
Let's add a comment in the description of the flag --num_striped_disks to emphasize this is: 1) for striping DATA disks, 2) all stripped data disk will show up as one single logical data disk and is indicated as data_disk_0 in the metadata.

Ideally I'd like to rename the flag to --num_striped_data_disks but I am ok if this is out of scope for this CL.